css/style.css

```diff
@@ -0,0 +1,750 @@
+  color: #344563;
+  padding-top: 2rem;
+}
+
+.links {
+  display: flex;
+  gap: 2em;
+  padding-left: 0;
+  margin-top: 22px;
+}
+
+.links > li {
+  list-style: none;
+}
+
+.tweeter,
+.Linkd,
+.cat {
+  margin: 0 20px 0 0;
+}
+
+#textarea {
+  margin: 0% 0% 4% 15%;
+  width: 69%;
+  height: 117px;
+  border: 1px solid #cfd8dc;
+  border-radius: 2px;
+  padding: 17px 0 0 17px;
+}
+
+.programming {
+  display: flex;
+  justify-content: space-between;
+}
+
+.programming img {
+  margin: 3px;
+  margin: 19px 19px 0 0;
+}
+
+.programming p {
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+}
+
+#name {
+  margin: 0% 0% 4% 15%;
+  width: 69%;
+  height: 48px;
+  border: 1px solid #cfd8dc;
+  border-radius: 2px;
+  padding: 0 0 0 17px;
+}
+
+#email {
+  margin: 0% 0% 4% 15%;
+  width: 69%;
+  height: 48px;
+  border: 1px solid #cfd8dc;
+  border-radius: 2px;
+  padding: 0 0 0 17px;
+}
+
+#email::placeholder,
+#name::placeholder {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+#textarea ::placeholder {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+}
+
+#submit {
+  margin: 0% 0 5% 29%;
+  padding: 12px 16px;
+  width: 140px;
+  background: #fff;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border: none;
+  cursor: pointer;
+}
+
+button:hover,
+#submit:hover {
+  background-color: #396df2;
+  color: #eee;
+}
+
+button:active,
+#submit:active {
+  background-color: blue;
+  border-color: #333;
+  color: #eee;
+}
+
+.buttonlanguage {
+  width: 138px;
+  height: 50px;
+  font-family: 'Poppins', Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  padding: 0 0 0 8px;
+}
+
+.contactpart {
+  display: inline;
+  align-items: center;
+}
+
+.connect {
+  padding-bottom: 1em;
+}
+
+.contactform {
+  width: 56%;
+  margin: auto;
+  align-items: center;
+  margin-top: 67px;
+  padding-top: 150px;
+}
+
+.contactform h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+}
+
+.contactform p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  margin-bottom: 30px;
+  color: #fff;
+}
```

Comment:
- [ ] Please kindly work more on the width of this paragraph as well. You can do it, be encouraged. 💪 

## Your design 

![A5](https://user-images.githubusercontent.com/103386183/199672294-5f863262-c85b-4f20-98a7-3d8cc560d306.png)

## Figma design

![A6](https://user-images.githubusercontent.com/103386183/199672354-c476be98-89bc-42ca-9dba-57093726fd23.png)

Reply:
## issue resolved 
Thank you for making that part clear for me 
-i reduced the width of that part 
-and center-aligned the text